src/transformers/models/t5/modeling_t5.py

```diff
@@ -181,7 +181,7 @@ def load_tf_weights_in_t5(model, config, tf_checkpoint_path):
     it will evenly distribute blocks across all devices.
 
     Args:
-        device_map (`Dict[int, list]`, optional, defaults to None):
+        device_map (`Dict[int, list]`, *optional*, defaults to None):
```

Comment:
We don't specify if it defaults to None

```suggestion
        device_map (`Dict[int, list]`, *optional*):
```